src/Examples/Program.cs

```diff
@@ -151,12 +152,13 @@ public static async void Mulitplayer(ZGame game)
             service.StartReceiving();
             resetEvent.WaitOne();
 
-            Console.WriteLine($"ID |  NAME :        | MAP :          |   Players    |");
+            Console.ForegroundColor = ConsoleColor.DarkYellow;
+            Console.WriteLine("\n {0,88} \n {1,5} {2,50}| {3,20}| {4,5}| \n {5,88}",strainthline,"ID:", "SERVERNAME" , "MAP:" ,"Players",strainthline);
             foreach (var item in service.ServersCollection)
             {
-                Console.WriteLine(string.Join("|",item.Id,item.Name,item.MapRotation.Current.Name,item.CurrentPlayersNumber));
+                Console.WriteLine("{0,5}| {1,50}| {2,20}| {3,2} / {4,2} |",item.Id,item.Name,item.MapRotation.Current.Name,item.CurrentPlayersNumber,item.PlayersCapacity);
```

Comment:
@nintend01337 Please remove Players column from server list output. It does not carry any semantic load there, since at this stage, we cannot ensure the loading of the players before the server list is drawn.